group_project_v2/public/js/components/submission.js

```diff
@@ -14,6 +16,24 @@ function GroupProjectSubmissionBlock(runtime, element) {
         $(document).trigger('group_project_v2.submission.upload_complete', uploadXHR);
     }
 
+    var message_box = $(".message"); // searching globally - not a typo: message box is created at group project level
```

Comment:
Any chance there's a cleaner way to do this? I know it's unlikely to ever be an issue in practice, but it would be nice to be able to scope this somehow.